### ISSUE TITLE
Selecting text with the keyboard

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#84]
```
Apple's Terminal.app allows text to be selected using the keyboard, which
is great for the mouse-phobic. To do so, text selection mode has to be
activated using the key combination Command+Option+Enter, after which the
cursor can be moved around the screen and scrollback to select the required
bit.

That would be a good feature to have in MinTTY, except for the three-key
shortcut perhaps. (I'd have thought those are illegal on Macs?)

Fullscreen could be bumped off Alt+Enter (to some Alt+Fkey combination?) to
make way for this.
```

Original issue reported on code.google.com by `andy.koppe` on 4 Apr 2009 at 7:47

**Comment by mintty** [comment]
Released 2.9.7.

**Comment by mintty** [comment]
Released 3.1.7.
